### PR TITLE
Update boot_folder.md

### PR DESCRIPTION
**Pull request by JamesH65**
Added table of kernel names/uses

### CODE REVIEW

**Comment by JamesH65** [comment]
Not convinced I have the table correct.

File: configuration/boot_folder.md [hunk 1]
```diff
@@ -47,6 +47,17 @@ This is the file to configure wireless network settings (if the hardware is capa
...
 
+### Kernel Files
+
+The boot folder will contain various kernel image files, used for the different Raspberry Pi models
+
+| Filename | Processor | Raspberry Model | Notes |
+| ---------| ----------|-----------------|-------|
+| kernel.img | BCM2835 | Pi 0, Pi 1, Pi2 | |
+| kernel7.img| BCM2836, BCM2837 | Pi2's, Pi3 | Later Pi2's used the BCM2837 |
+| kernel7l.img | BCM2711 | Pi 4 | Large Pointer Address Extensions |
```
Comment (pelwell) [inline configuration/boot_folder.md]: Large _Physical_ Address _Extension_

File: configuration/boot_folder.md [hunk 2]
```diff
@@ -47,6 +47,17 @@ This is the file to configure wireless network settings (if the hardware is capa
 
 There are various Device Tree blob files, which have the extension `.dtb`. These contain the hardware definitions of the various models of Raspberry Pi, and are used on boot to set up the kernel according to which Pi model is detected. More [details here](device-tree.md).
 
+### Kernel Files
+
+The boot folder will contain various kernel image files, used for the different Raspberry Pi models
```
Comment (pelwell) [inline configuration/boot_folder.md]: End with a colon.

**Comment by burtyb** [comment]
Should the Pi2 be on the kernel.img/BCM2835 row?

**Comment by pelwell** [comment]
No, we'll try kernel7.img first, but the firmware will switch to "older" names if the "newer" ones aren't found.

File: configuration/boot_folder.md [hunk 3]
```diff
@@ -49,13 +49,13 @@ There are various Device Tree blob files, which have the extension `.dtb`. These
...
 
-The boot folder will contain various kernel image files, used for the different Raspberry Pi models
+The boot folder will contain various kernel image files, used for the different Raspberry Pi models:
 
 | Filename | Processor | Raspberry Model | Notes |
 | ---------| ----------|-----------------|-------|
 | kernel.img | BCM2835 | Pi 0, Pi 1, Pi2 | |
 | kernel7.img| BCM2836, BCM2837 | Pi2's, Pi3 | Later Pi2's used the BCM2837 |
-| kernel7l.img | BCM2711 | Pi 4 | Large Pointer Address Extensions |
+| kernel7l.img | BCM2711 | Pi 4 | Large Physical Address Extensions |
```
Comment (pelwell) [inline configuration/boot_folder.md]: Extension. not Extensions

File: configuration/boot_folder.md [hunk 4]
```diff
@@ -47,6 +47,17 @@ This is the file to configure wireless network settings (if the hardware is capa
 
 There are various Device Tree blob files, which have the extension `.dtb`. These contain the hardware definitions of the various models of Raspberry Pi, and are used on boot to set up the kernel according to which Pi model is detected. More [details here](device-tree.md).
 
+### Kernel Files
+
+The boot folder will contain various kernel image files, used for the different Raspberry Pi models:
+
+| Filename | Processor | Raspberry Model | Notes |
+| ---------| ----------|-----------------|-------|
+| kernel.img | BCM2835 | Pi 0, Pi 1, Pi2 | |
```
Comment (lurch) [inline configuration/boot_folder.md]: There were no Pi2s with the BCM2835 :stuck_out_tongue_winking_eye:

File: configuration/boot_folder.md [hunk 5]
```diff
@@ -47,6 +47,17 @@ This is the file to configure wireless network settings (if the hardware is capa
...
+| kernel7.img| BCM2836, BCM2837 | Pi2's, Pi3 | Later Pi2's used the BCM2837 |
+| kernel7l.img | BCM2711 | Pi 4 | Large Physical Address Extension |
+| kernel8.img  | BCM2836, BCM2836, BCM2711 | Pi2/3/4 | Beta 64 bit kernel|
```
Comment (lurch) [inline configuration/boot_folder.md]: BCM2836, BCM2836 -> BCM2836, BCM2837

File: configuration/boot_folder.md (hunk 4 above)
Comment (pelwell) [reply configuration/boot_folder.md]: Aaagh - yes remove it.

File: configuration/boot_folder.md (hunk 5 above)
Comment (lurch) [reply configuration/boot_folder.md]: Maybe this should also cross-link to the config.txt section where it explains how this 64-bit kernel is selected?

**Comment by pelwell** [comment]
Sorry, @burtyb I read your comment with the wrong emphasis - you were, of course, suggesting its removal rather than its insertion.

**Comment by pelwell** [comment]
Maybe cross-linking etc. can be done in another commit once the technical information is correct?

**Pull request merged**